src/components/Attachments/ImageAttachment.js

```diff
@@ -2,18 +2,17 @@ import { Box } from '@rocket.chat/fuselage';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ImageAttachment = ({ attachment }) => {
-  return (
-    <Box>
-      <p>{attachment?.description}</p>
-      <img
-        src={'http://localhost:3000' + attachment.image_url}
-        height={attachment.image_dimensions.height}
-        width={attachment.image_dimensions.width}
-      />
-    </Box>
-  );
-};
+const ImageAttachment = ({ attachment }) => (
+  <Box>
+    <p>{attachment?.description}</p>
+    <img
+      src={`http://localhost:3000${attachment.image_url}`}
+      height={attachment?.image_dimensions?.height ?? 357}
+      width={attachment?.image_dimensions?.width ?? 476}
```

Comment:
Is it responsive while we use it width let's say less than this?

Reply:
Yes, it will be.
But less than this width won't be there in the common case.